spec/models/submission_spec.rb

```diff
@@ -886,4 +887,28 @@
       end
     end
   end
+
+  describe "#final_submission_feedback_files?" do
+    it 'returns true if at least one admin feedback file with type final-submission' do
+      submission.admin_feedback_files.build(feedback_type: 'final-submission')
+      expect(submission).to be_final_submission_feedback_files
+    end
+
+    it 'returns false if no admin feedback file has the type of final-submission' do
+      submission.admin_feedback_files.build(feedback_type: 'not final-submission')
+      expect(submission).not_to be_final_submission_notes
+    end
+  end
+
+  describe "#format_review_feedback_files?" do
+    it 'returns true if at least one admin feedback file with type format-review' do
+      submission.admin_feedback_files.build(feedback_type: 'format-review')
+      expect(submission).to be_final_submission_feedback_files
+    end
+
+    it 'returns false if no admin feedback file has the type of format-review' do
+      submission.admin_feedback_files.build(feedback_type: 'not format-review')
+      expect(submission).not_to be_final_submission_notes
```

Comment:
This too: `be_final_submission_feedback_files`?

Reply:
Good catch!